python/ray/function_manager.py

```diff
@@ -444,7 +444,14 @@ def get_execution_info(self, driver_id, function_descriptor):
         # the function from GCS.
         with profiling.profile("wait_for_function", worker=self._worker):
             self._wait_for_function(function_descriptor, driver_id)
-        return self._function_execution_info[driver_id][function_id]
+        try:
+            info = self._function_execution_info[driver_id][function_id]
+        except KeyError as e:
+            message = ("Error occurs in get_execution_info: "
+                       "driver_id: %s, function_descriptor: %s. Message: %s" %
+                       (binary_to_hex(driver_id), function_descriptor, e))
+            raise KeyError(message)
```

Comment:
If a user called a function that does not belong to an actor, this KeyError will happen. However, in this scenario, this the key error comes from Function ID instead of Driver ID.